Makefile

```diff
@@ -789,41 +780,19 @@ $(PROGRAM_PREFIX)yosys-config: misc/yosys-config.in
 			-e 's#@BINDIR@#$(strip $(BINDIR))#;' -e 's#@DATDIR@#$(strip $(DATDIR))#;' < $< > $(PROGRAM_PREFIX)yosys-config
 	$(Q) chmod +x $(PROGRAM_PREFIX)yosys-config
 
-abc/abc-$(ABCREV)$(EXE) abc/libabc-$(ABCREV).a:
+ABC_SOURCES := $(wildcard $(YOSYS_SRC)/abc/*)
+
+
+# This is a relative path, that is, we would like to build abc in $ROOT/abc/abc if compiling from root or $ROOT/build/abc/abc if compiling from $ROOT/build
```

Comment:
This seems a bit of a confusing comment, as it appears to imply there's no support for out-of-tree builds.